src/main/java/org/itsallcode/openfasttrace/report/PlainTextReport.java

```diff
@@ -45,6 +44,8 @@
     private static final Comparator<LinkedSpecificationItem> LINKED_ITEM_BY_ID = Comparator
             .comparing(LinkedSpecificationItem::getId);
     private final Newline newline;
+    private ReportVerbosity verbosity;
```

Comment:
Maybe set default verbosity, e.g. null? Or set verbosity in constructor and make it final?

Reply:
Verbosity is now not a class member anymore. This way it can't be uninitialized (except if the caller does not initialize it.